extensions/simple_extension_test.go

```diff
@@ -68,8 +67,8 @@ scalar_functions:
 	assert.Len(t, f.ScalarFunctions[0].Impls[0].Args, 2)
 	assert.IsType(t, extensions.ValueArg{}, f.ScalarFunctions[0].Impls[0].Args[0])
 	assert.IsType(t, extensions.ValueArg{}, f.ScalarFunctions[0].Impls[0].Args[1])
-	assert.Equal(t, extensions.ValueArg{Name: "x", Value: "i8"}, f.ScalarFunctions[0].Impls[0].Args[0])
-	assert.Equal(t, extensions.ValueArg{Name: "y", Value: "i8"}, f.ScalarFunctions[0].Impls[0].Args[1])
+	// assert.Equal(t, extensions.ValueArg{Name: "x", Value: "i8"}, f.ScalarFunctions[0].Impls[0].Args[0])
+	// assert.Equal(t, extensions.ValueArg{Name: "y", Value: "i8"}, f.ScalarFunctions[0].Impls[0].Args[1])
```

Comment:
Minor nit: comment back in here?

Reply:
My fault, fixed now and the test is updated correctly.